python/jsbeautifier/tests/shell-smoke-test.sh

```diff
@@ -77,31 +77,36 @@ test_cli_js_beautify()
       exit 1
   }
 
-  rm -rf /tmp/js-beautify-mkdir
-  $CLI_SCRIPT -o /tmp/js-beautify-mkdir/js-beautify.js $SCRIPT_DIR/../../../js/bin/js-beautify.js && diff $SCRIPT_DIR/../../../js/bin/js-beautify.js /tmp/js-beautify-mkdir/js-beautify.js || {
-      echo "js-beautify output for $SCRIPT_DIR/../../../js/bin/js-beautify.js should have been created in /tmp/js-beautify-mkdir/js-beautify.js."
+  JS_BEAUTIFY_MKDIR=`mktemp --tmpdir -d js-beautify-mkdir-XXXX`
+  $CLI_SCRIPT -o $JS_BEAUTIFY_MKDIR/js-beautify.js $SCRIPT_DIR/../../../js/bin/js-beautify.js && diff $SCRIPT_DIR/../../../js/bin/js-beautify.js $JS_BEAUTIFY_MKDIR/js-beautify.js || {
+      echo "js-beautify output for $SCRIPT_DIR/../../../js/bin/js-beautify.js should have been created in $JS_BEAUTIFY_MKDIR/js-beautify.js."
       exit 1
   }
 
   # ensure unchanged files are not overwritten
-  cp -p /tmp/js-beautify-mkdir/js-beautify.js /tmp/js-beautify-mkdir/js-beautify-old.js
-  touch -A -05 /tmp/js-beautify-mkdir/js-beautify.js
-  touch -A -01 /tmp/js-beautify-mkdir/js-beautify-old.js
-  $CLI_SCRIPT -r /tmp/js-beautify-mkdir/js-beautify.js && test /tmp/js-beautify-mkdir/js-beautify.js -nt /tmp/js-beautify-mkdir/js-beautify-old.js && {
-      echo "js-beautify should not replace unchanged file /tmp/js-beautify-mkdir/js-beautify.js when using -r"
+  cp -p $JS_BEAUTIFY_MKDIR/js-beautify.js $JS_BEAUTIFY_MKDIR/js-beautify-old.js
+  touch -A -05 $JS_BEAUTIFY_MKDIR/js-beautify.js
+  touch -A -01 $JS_BEAUTIFY_MKDIR/js-beautify-old.js
+  $CLI_SCRIPT -r $JS_BEAUTIFY_MKDIR/js-beautify.js && test $JS_BEAUTIFY_MKDIR/js-beautify.js -nt $JS_BEAUTIFY_MKDIR/js-beautify-old.js && {
+      echo "js-beautify should not replace unchanged file $JS_BEAUTIFY_MKDIR/js-beautify.js when using -r"
+      rm -rf $JS_BEAUTIFY_MKDIR
```

Comment:
Instead of adding this multiple times, could you move this and the `exit 1` into a function?